### ISSUE TITLE
One colour in accessible colour set is not accessible

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#1138]
Non-text contrast guidelines require a colour contrast ratio of 3:1; the blue #55a6e5 line fails this against the white background at 2.6:1

See here:
http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/1.3.0-dev/1-2-1/

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning @phillipgardiner this was the main feedback from DAC test of new chart colours

**Comment by brockfanning** [comment]
So that's the first disaggregation color, right? @phillipgardiner any ideas on what we should change it to?

**Comment by phillipgardiner** [comment]
Umm. Not sure how that happened! yes leave it with me and I will come back with an alternative.

**Comment by brockfanning** [comment]
@phillipgardiner To aid discussion here are the colors/orders as things stand now:

Headline - normal:

![#004466](https://www.thecolorapi.com/id?hex=004466&format=svg)

Headline - high contrast:

![#FFDD00](https://www.thecolorapi.com/id?hex=FFDD00&format=svg)

Disaggregations:

1. ![#55a6e5](https://www.thecolorapi.com/id?hex=55a6e5&format=svg)
2. ![#8d4d57](https://www.thecolorapi.com/id?hex=8d4d57&format=svg)
3. ![#cd7a00](https://www.thecolorapi.com/id?hex=cd7a00&format=svg)
4. ![#993300](https://www.thecolorapi.com/id?hex=993300&format=svg)
5. ![#339966](https://www.thecolorapi.com/id?hex=339966&format=svg)
6. ![#9966cc](https://www.thecolorapi.com/id?hex=9966cc&format=svg)

**Comment by brockfanning** [comment]
@phillipgardiner Do I understand right that the change should be:

Headline - normal:

![#004466](https://www.thecolorapi.com/id?hex=004466&format=svg)

Headline - high contrast:

![#55a6e5](https://www.thecolorapi.com/id?hex=55a6e5&format=svg)

Disaggregations:

1. ![#8d4d57](https://www.thecolorapi.com/id?hex=8d4d57&format=svg)
2. ![#cd7a00](https://www.thecolorapi.com/id?hex=cd7a00&format=svg)
3. ![#993300](https://www.thecolorapi.com/id?hex=993300&format=svg)
4. ![#339966](https://www.thecolorapi.com/id?hex=339966&format=svg)
5. ![#9966cc](https://www.thecolorapi.com/id?hex=9966cc&format=svg)
6. ![#FFDD00](https://www.thecolorapi.com/id?hex=FFDD00&format=svg)

**Comment by phillipgardiner** [comment]
@brockfanning Thank you. As discussed on the tech call, I propose the following colours to be used, in this order:

Normal contrast: #004466 (Headline), #cd7a00, #339966, #9966cc, #8d4d57, #993300, #ffdd00

High contrast: #55a6e5 (Headline), #cd7a00, #339966, #9966cc, #8d4d57, #993300, #ffdd00

Hope that makes sense? Please could you drop me a screen shot of the default line on high contrast as just interest to see if it looks right.

**Comment by phillipgardiner** [comment]
@brockfanning looks like our comments overlapped. Yes you got it :)

**Comment by phillipgardiner** [comment]
@brockfanning assuming Yes. The order would be:

https://camo.githubusercontent.com/301760c5ca74771f429cc5efe7b97d649b461a4e7ff574dd8c81e3348e8e3836/68747470733a2f2f7777772e746865636f6c6f726170692e636f6d2f69643f6865783d38643464353726666f726d61743d737667

https://camo.githubusercontent.com/a45a3ee0419985fc1a53f9868e49d06edf13d735a642e68b44a866d7677a498f/68747470733a2f2f7777772e746865636f6c6f726170692e636f6d2f69643f6865783d33333939363626666f726d61743d737667

https://camo.githubusercontent.com/2075793282f816104f657b41d474a50739967e733d0ba69c554eb54528d6dff7/68747470733a2f2f7777772e746865636f6c6f726170692e636f6d2f69643f6865783d63643761303026666f726d61743d737667

https://camo.githubusercontent.com/6cb44c76d85c1c11121fb6fcf198443ee44fbacdf041aaaa3a7fc0de98f20e8a/68747470733a2f2f7777772e746865636f6c6f726170692e636f6d2f69643f6865783d39393636636326666f726d61743d737667

https://camo.githubusercontent.com/43dc88ee5b0e7698b21566e6e6b5929ff34c43e2db4ab833b8f7b2a8b2d90d4f/68747470733a2f2f7777772e746865636f6c6f726170692e636f6d2f69643f6865783d39393333303026666f726d61743d737667

https://camo.githubusercontent.com/38e1366e5695d4e11d646650af33037ca7e249fc2a1bfbc3472495dec4b12ae0/68747470733a2f2f7777772e746865636f6c6f726170692e636f6d2f69643f6865783d46464444303026666f726d61743d737667

**Comment by phillipgardiner** [comment]
@brockfanning any questions please shout :)

**Comment by brockfanning** [comment]
@phillipgardiner To confirm about the order of the disaggregations, it should be like this?

1. ![#8d4d57](https://www.thecolorapi.com/id?hex=8d4d57&format=svg)
4. ![#339966](https://www.thecolorapi.com/id?hex=339966&format=svg)
2. ![#cd7a00](https://www.thecolorapi.com/id?hex=cd7a00&format=svg)
5. ![#9966cc](https://www.thecolorapi.com/id?hex=9966cc&format=svg)
3. ![#993300](https://www.thecolorapi.com/id?hex=993300&format=svg)
6. ![#FFDD00](https://www.thecolorapi.com/id?hex=FFDD00&format=svg)

**Comment by phillipgardiner** [comment]
@brockfanning Yeah perfect - thank you

**Comment by brockfanning** [comment]
@phillipgardiner I just checked https://webaim.org/resources/contrastchecker/ and if I'm reading it right, #FFDD00 over #FFFFFF only has a 1.34:1 ratio. Is that going to be an issue?

**Comment by phillipgardiner** [comment]
@brockfanning gggrrr what am I doing! Yes that an issue as not accessible. Sorry, leave it with me a bit.

**Comment by phillipgardiner** [comment]
@brockfanning OK, I think I have an answer.....involves a couple of swaps

Change:
#004466 to #00006A (normal contrast headline)
#55a6e5 to #FFDD00 (high contrast headline) - so we are going back to using yellow for the high contrast headline colour
then for both
#FFDD00 to #054CE6
#993300 to #A33600

So to clarify, we should now have:

#00006a (headline), #cd7a00, #339966, #9966cc, #8d4d57, #993300, #054ce6

#FFDD00 (headline), #cd7a00, #339966, #9966cc, #8d4d57, #993300, #054ce6

I'm really hoping that all works!!!!! sorry for delay.
